src/main/java/edu/hawaii/its/api/groupings/GroupingSyncDestinations.java

```diff
@@ -52,10 +72,20 @@ private void setSyncDestination(List<AttributesResult> attributesResults) {
             groupingSyncDestination.setSynced(this.groupAttributes.stream()
                     .anyMatch(groupAttribute -> groupAttribute.getAttributeName().equals(attributesResult.getName())));
             this.syncDestinations.add(groupingSyncDestination);
+            this.syncDestinations.sort(Comparator.comparing(GroupingSyncDestination::getDescription));
         }
     }
 
     private void setGroupAttributes(List<GroupAttribute> groupAttributes) {
         this.groupAttributes = groupAttributes;
     }
+    private void parseKeyVal(String GOOGLE_SYNC_DEST_SUFFIX) {
+        this.syncDestinations.forEach(dest -> {
+            String tmpDescription = dest.getDescription().replaceAll("\\$\\{srhfgs}", groupExtension);
+            if (dest.getName().contains("google-group")) {
+                tmpDescription = tmpDescription + GOOGLE_SYNC_DEST_SUFFIX;
+            }
+            dest.setDescription(tmpDescription);
+        });
+    }
```

Comment:
I think this parseKeyVal function would be best called in setSyncDestinations rather than the constructor.